tests/connectors/kubernetes_test.py

```diff
@@ -1588,9 +1587,6 @@ async def test_adjust_tuning_settlement_failed(
         with pytest.raises(AdjustmentRejectedError) as rejection_info:
             await connector.adjust([adjustment])
 
-        # Validate raised warnings to ensure all coroutines were awaited
-        assert not any(filter(lambda warn: "was never awaited" in warn.message, recwarn)), list(map(lambda warn: warn.message, recwarn))
-
```

Comment:
That error checking logic is now redundant since the pytest configuration was [updated to treat all warnings as errors](https://github.com/opsani/servox/blob/31197324a2d1cb7e1124f0a8f4665bba56d7b396/setup.cfg#L5).

Further, it caused an error during a test run that couldn't be debugged (perhaps due to conflicting logic with the change above) so I advised that the check be removed to let the generic warning handling catch this warning instead: https://github.com/opsani/servox/runs/3352963416